src/Operations.jl

```diff
@@ -1159,21 +1156,22 @@ function rm(ctx::Context, pkgs::Vector{PackageSpec})
     # delete drops from project
     n = length(ctx.env.project["deps"])
     filter!(ctx.env.project["deps"]) do (_, uuid)
-        UUID(uuid) ∉ drop
+        uuid ∉ drop
     end
     if length(ctx.env.project["deps"]) == n
         @info "No changes"
         return
     end
     deps_names = collect(keys(ctx.env.project["deps"]))
-    if haskey(ctx.env.project, "targets")
-        filter!(ctx.env.project["targets"]) do (target, deps)
-            !isempty(filter!(in(deps_names), deps))
-        end
-        if isempty(ctx.env.project["targets"])
-            delete!(ctx.env.project, "targets")
-        end
+    filter!(ctx.env.project["targets"]) do (target, deps)
+        !isempty(filter!(in(deps_names), deps))
+    end
+    #= WHY delete?
+    if isempty(ctx.env.project["targets"])
+        delete!(ctx.env.project, "targets")
```

Comment:
Oh, is this #876?

Reply:
Although I don't know why it would be empty.

Reply:
I don't think so; the "targets" section wouldn't be empty in that case, right?

Reply:
I think this can just be deleted.

Reply:
I think I was thinking you could `rm` a Test dependency and when the target section got empty we should remove the section.

Reply:
Maybe something accidentally removed it from targets, and then this line removed the targets completely.